pkg/resources/reconciler_test.go

```diff
@@ -579,3 +607,20 @@ func TestReconciler_ReconcileNamespace(t *testing.T) {
 		})
 	}
 }
+
```

Comment:
We can remove this if it won't be added.

Reply:
Sorted below, thanks